RELEASE.md

```diff
@@ -0,0 +1,31 @@
+## Major / Minor Release
+
+- create a branch for the release e.g. `7.6.x`
+- provide a [PR](https://github.com/eclipse/che-machine-exec/pull/66) with bumping the [VERSION](https://github.com/eclipse/che-machine-exec/blob/master/VERSION) file to the `7.6.x` branch
+- [![Release Build Status](https://ci.centos.org/buildStatus/icon?subject=release&job=devtools-che-machine-exec-release/)](https://ci.centos.org/job/devtools-che-machine-exec-release/) CI is triggered based on the changes in the [release](https://github.com/eclipse/che-machine-exec/tree/release) branch (not `7.6.x`).
+
+In order to trigger the CI once the [PR](https://github.com/eclipse/che-machine-exec/pull/66) is merged to the `7.6.x` one needs to:
+
+```
+ git fetch origin 7.6.x:7.6.x
+ git checkout 7.6.x
+ git branch release -f 
+ git push origin release -f
+```
+
```

Comment:
Are the above 4 steps done by a bot or a human? If a human, do we have a che-bot that has permission to push to that branch? And does that branch support forced pushes w/o PR checks?

Reply:
> Are the above 4 steps done by a bot or a human? If a human, do we have a che-bot that has permission to push to that branch that we could use instead?

Currently, those steps are manual but of course this should be automated somehow in future

> And does that branch support forced pushes w/o PR checks?

yes, `release` branch support force push